library/Garp/Mail/Transport/AmazonSes.php

```diff
@@ -13,38 +13,11 @@
 class Garp_Mail_Transport_AmazonSes extends Zend_Mail_Transport_Abstract
```

Comment:
Pas het docblock hier ook maar aan — ik heb deze code ooit overgenomen van internet, maar inmiddels is het niet echt meer die code natuurlijk. 🙂